website/docs/components/alert/partials/code/how-to-use.md

```diff
@@ -70,6 +70,10 @@ The default `@tag` is `"div"` because the correct value is dependent on the indi
 
 A different color can be applied to the Alert using the `color` argument. This will determine the default icon used in the Alert, unless overwritten.
 
+The “success”, “warning”, and “critical” Alert colors will be read to users before any other content as they include either `role="alert"` or `role="alertdialog"` and `aria-live="polite"` attributes by default.
+
+For informational or promotional content which is non-critical to users, use either the “neutral” or “highlight” colors as these do not include `role` or `aria-live` attributes.
+
```

Comment:
I have taken a little time to think about this and write up some suggested text. What do you think?

```suggestion
The available color values are `neutral` (the default), `highlight`, `success`, `warning`, and `critical`. Setting a color value will also determine the default icon used in the Alert, although it is customizable. 

The color value will also determine some accessibility features of the component, so this should be taken into consideration when choosing which Alert `color` value to use.

If the alert is being used in an informational or promotional way, `neutral` or `highlight` colors should be chosen. 

The other color values map to accessibility-related roles, and will ensure that essential information is presented to the user with assistive technology in a timely manner. 
```

Reply:
Looks good, thanks!